### ISSUE TITLE
error with the CLI 

### ISSUE THREAD

**Issue by arademaker** [FredsoNerd/py-ownpt#9]
```
python -m pyownpt.cli.update data/own-pt-* --wns dump/wn.jsonl --vts dump/votes.jsonl --sgs dump/suggestion-* -l pt -u arademaker vcvpaiva -o own-pt.nt -v
/Users/ar/venv/bin/python: Error while finding module specification for 'pyownpt.cli.update' (ModuleNotFoundError: No module named 'pyownpt.cli')
...
```

when I run in the release of OWN-PT and OWN-EN...

**Comment by arademaker** [comment]
Please @FredsoNerd need urgent support.

**Comment by fredsonaguiar** [comment]
You're right. In a moment, we removed the sub-directory `pyownpt.cli` from installation. I've just fixed it in 607c60cdacdd47b482ebf10856c80e5bafdc16be. Please, reinstall the package or install it in a new environment.

**Comment by arademaker** [comment]
version is still 0.0.0

```
Successfully installed Py-OWNPT-0.0.0 certifi-2021.5.30 charset-normalizer-2.0.4 idna-3.2 isodate-0.6.0 pyparsing-2.4.7 rdflib-6.0.0 rdflib-jsonld-0.5.0 requests-2.26.0 six-1.16.0 tabulate-0.8.9 toml-0.10.2 tqdm-4.62.1 urllib3-1.26.6 wn-0.8.0
```
